cli/tsc/00_typescript.js

```diff
@@ -149187,7 +149187,7 @@ ${newComment.split("\n").map((c) => ` * ${c}`).join("\n")}
               break;
             case 3 /* CommonJS */:
             case 2 /* Namespace */:
-              Debug.assert(entry.namespaceLikeImport === void 0 || entry.namespaceLikeImport.name === symbolName2, "Namespacelike import shoudl be missing or match symbolName");
+              Debug.assert(entry.namespaceLikeImport === void 0 || entry.namespaceLikeImport.name === symbolName2, "Namespacelike import should be missing or match symbolName");
```

Comment:
```suggestion
              Debug.assert(entry.namespaceLikeImport === void 0 || entry.namespaceLikeImport.name === symbolName2, "Namespacelike import shoudl be missing or match symbolName");
```

Let's revert this one because it's in the typescript code.

Reply:
Also, when opening PRs, please open from a branch on the fork instead of committing directly to the main branch. I can't commit this change and merge this PR right away because it was opened from the main branch (it's also easier to manage changes on a fork when doing so)